VMBackup/main/Utils/DiskUtil.py

```diff
@@ -327,9 +340,12 @@ def get_mount_file_systems(self):
                                 fstypeEnd = line.find(" ", fstypeStart+1)
                                 if(fstypeEnd >=0):
                                     fs_type = line[fstypeStart+1:fstypeEnd]
-                            # If there is a duplicate, keep only the first instance
-                    if (file_system,fs_type,mount_point) not in file_systems_info:
+                    # If there is a duplicate, keep only the first instance
+                    if ((mount_point not in mount_points) and ((file_system,fs_type,mount_point) not in file_systems_info)):
```

Comment:
Suggestion :Same here, older check should suffice.

Reply:
As commented above keeping only (mount_point not in mount_points)